pyang/error.py

```diff
@@ -175,12 +175,12 @@ def __init__(self, msg="", exit_code=1):
     'NODE_NOT_FOUND':
       (1,
        'node %s::%s is not found'),
-    'NODE_NOT_FOUND1':
-      (1,
-       'node %s::%s is not found in %s::%s'),
-    'NODE_NOT_FOUND2':
-      (1,
-       'node %s::%s is not found in module %s'),
+    #'NODE_NOT_FOUND1':
```

Comment:
These can be removed; they were added by mistake (or so it seems when I check the commit where they were added).
I can remove them after this merge, you don't have to update the PR.